operations/update.go

```diff
@@ -53,96 +59,116 @@ func Update() cli.Command {
 			if err != nil {
 				return errors.Wrap(err, "problem loading configuration")
 			}
-
-			client := conf.getRestCommunicator(ctx)
-			defer client.Close()
-
-			update, err := checkUpdate(client, false, forceUpdate)
-			if err != nil {
-				return err
+			if !conf.AutoUpgradeCLI && !autoUpgrade {
```

Comment:
Do we want to do this always, or only when there is an update available?

Reply:
I think doing it always is reasonable since this is the update command.